modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -1348,7 +1364,22 @@ private MediaPackage mergeScheduledMediaPackage(MediaPackage mp) throws IngestEx
     }
   }
 
-  private void mergeMediaPackageElements(MediaPackage mp, MediaPackage scheduledMp) {
+  private void mergeMediaPackageElements(final MediaPackage mp, final MediaPackage scheduledMp) {
+    // drop catalogs sent by the capture agent in favor of Opencast's own metadata
+    if (skipCatalogs) {
```

Comment:
We do this with the elements, do we also need to do this with the metadata?  In other words, does `mergeMediaPackageMetadata` also needs these changes?

Reply:
Done.